server.js

```diff
@@ -1,16 +1,31 @@
 const express = require("express");
 const cors = require("cors");
-
+const uuid = require("uuid");
+const bodyParser = require("body-parser");
 const app = express();
+const PORT = process.env.PORT || 3000;
+
+// create application/x-www-form-urlencoded parser - middlleware function
+const urlencodedParser = bodyParser.urlencoded({ extended: false });
 
 app.use(cors());
+app.use(express.json());
 
 const welcomeMessage = {
   id: 0,
   from: "Bart",
   text: "Welcome to CYF chat system!",
 };
 
+// Functions
+function isInvalidId(id, index, response) {
+  if (index < 0) {
+    response.status(400).json({
+      msg: "No message with the Id '" + id + "' is found",
+    });
+  }
+}
```

Comment:
I like the code reuse, but I don't know how I feel about it from a code readability POV...it might be completely fine to do this in express-land